### DISCUSSION TITLE
change vector daemonset timezone

### DISCUSSION THREAD

**Discussion by pablofilgueira91** [vectordotdev/vector#20464] [Q&A]
### A note for the community

I am using vector to collect the logs of a service and save it in an Azure storage, this saves it in one folder per day.
The problem I have is that the vector time is in UTC since it is the time zone of the kubernetes cluster, and my time zone is UTC-3, which gives me the problem that it creates the folder for the next day and for 3 hours saves logs from the previous day.
Is there a way to set the correct time zone in vector?

**Comment by Alex-wwei** [comment]
It occured to me. Is it fixed?I am using vector to collect the logs of DB and save it in ES storage, this saves it in an index by day (vector-%Y.%m.%d).
